TaskManager/Home/View/DataSourceDiffable.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DataSourceDiffable.swift
+//  TaskManager
+//
+//  Created by user on 15.09.2022.
+//
+
+import Foundation
+import UIKit
+
+class DataSourceDiffable: UITableViewDiffableDataSource<Section, ModelTask> {
+    
+    let sectionArray = ["Active", "Completed"]
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return sectionArray[section]
+    }
+    
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+
+        let myLabel = UILabel()
+        myLabel.frame = CGRect(x: 20, y: 8, width: 320, height: 20)
```

Comment:
```suggestion
        let myLabel = UILabel.appearance(whenContainedInInstancesOf: [UITableViewHeaderFooterView.self])
```
try this will it work?